package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bigcommerce-cornerstone",
-  "version": "4.0.0-rc.2",
+  "version": "4.0.0",
```

Comment:
Something is wrong here .. your PR shouldn't have this change.

Reply:
Please ignore this comment. I have created a follow up ticket to fix this issue.